.circleci/config.yml

```diff
@@ -15,13 +15,15 @@ jobs:
       - setup_remote_docker
       - run:
           name: "Build Docker Images"
-          command: ./build-images.sh
+          command: |
+            ./build-images.sh
+            echo 'export DOCKER_PASS=$DOCKER_TOKEN' >> $BASH_ENV
```

Comment:
Do you still need this?

Reply:
Yes, this was added in this PR. The Stubb tool (as well as others) only understand Docker_pass so it's being set as an alias to Docker_token.

Reply:
sounds good